docs/08-developer-guide/01-build/01-maven.md

```diff
@@ -0,0 +1,129 @@
+---
+sidebar_label: Maven
+---
+
+<!-- cspell:words xzf Dskip Pdist installnpm installnpx installyarn -->
+
+# Building Ozone With Maven
+
+**TODO:** File a subtask under [HDDS-9861](https://issues.apache.org/jira/browse/HDDS-9861) and complete this page or section.
+
+- Cover basic Maven commands to build and run tests.
+- Document all the Ozone specific Maven flags we use to speed up or skip parts of the build, and when they are useful.
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+This guide explains how to build Apache Ozone from source using Maven and prepare it for deployment.
+
+## Prerequisites
+
+**TODO** : [HDDS-11625](https://issues.apache.org/jira/browse/HDDS-11625) Finalize the version numbers of prerequisite packages
+
+Before you begin, ensure you have the following installed on your build machine:
+
+- Java 1.8 or higher
+- Apache Maven 3.6.3 or higher
+- Git (if building from source repository)
+
+## Obtain the Source Code
+
+You can build Apache Ozone either by cloning the source code from Git or by downloading the official source tarball.
+
+Choose one of the following methods to get the source code:
+
+<Tabs>
+  <TabItem value="Git" label="Git" default>
+    Use this option to build any release, tag or commit of Ozone.
+    ```bash
+    git clone https://github.com/apache/ozone.git
+    cd ozone
+    ```
+  </TabItem>
+  <TabItem value="Tarball" label="Tarball">
+    Use this option if you only want the released version of Ozone.
+    <br/>Obtain the Ozone sources from the [download](/download) page.
+    <br/>Next, unpack the tarball
+    ```bash
+    tar xzf ozone-<version>-src.tar.gz
+    cd ozone-<version>-src
+    ```
+  </TabItem>
+</Tabs>
+
+## Build the Project
+
+Apache Ozone uses Maven as its build system. The build process compiles the source code, runs tests, and creates deployable artifacts. The project supports various build configurations to accommodate different development and deployment needs.
+
+### Build Options
+
+The build system offers several options to customize the build process according to your requirements:
+
+#### Basic Build
+
+For a basic build that skips tests:
+
+```bash
+mvn clean install -DskipTests=true
```

Comment:
I think we can change these commands back to `package` per [this discussion](https://github.com/apache/ozone-site/pull/100#discussion_r1830222607)